assemble/conf/accumulo.properties

```diff
@@ -31,3 +31,19 @@ instance.secret=DEFAULT
 
 ## Set to false if 'accumulo-util build-native' fails
 tserver.memory.maps.native.enabled=true
+
+## Set compaction settings
```

Comment:
Thinking ahead to the elasticity branch, this file could define single compaction service and have a link to the Accumulo docs with more complex recommended config for larger clusters.

Reply:
yeah with elasticity this becomes a single compaction service definition of three lines.